Dockerfile

```diff
@@ -0,0 +1,79 @@
+# Install dependencies only when needed
+FROM node:14 AS deps
+
+ARG TWOOPSTRACKER_API_URL
+ARG NEXT_PUBLIC_TWOOPSTRACKER_API_URL
+ARG NEXTAUTH_PROVIDERS_OAUTH_LOGIN_URL
+ARG NEXTAUTH_JWT_TOKEN_VERIFY_URL
+ARG NEXTAUTH_JWT_TOKEN_REFRESH_URL
+ARG NEXTAUTH_PROVIDERS_GOOGLE_CLIENT_SECRET
+ARG NEXTAUTH_PROVIDERS_GOOGLE_CLIENT_ID
+ARG NEXTAUTH_URL
+ARG NEXT_PUBLIC_IMAGE_DOMAINS
+ARG NEXTAUTH_SECRET
+ARG S3_UPLOAD_SECRET
+ARG S3_UPLOAD_KEY
+ARG S3_UPLOAD_BUCKET
+ARG S3_UPLOAD_REGION
+ARG ADMIN_BACKEND_REPO
+ARG ADMIN_BACKEND_AUTH_ENDPOINT
+ARG ADMIN_OAUTH_CLIENT_ID
+ARG ADMIN_OAUTH_CLIENT_SECRET
+ARG ADMIN_OAUTH_SCOPE
```

Comment:
```suggestion
ARG ADMIN_OAUTH_SCOPE

# Next.js collects completely anonymous telemetry data about general usage.
# Learn more here: https://nextjs.org/telemetry
ARG NEXT_TELEMETRY_DISABLED=1
```

Reply:
@kilemensi  I've noticed most of these ARGS aren't used during Build so I've removed them. Lemme know if there's something that seems like it's needed and it's not there.